javax-money/src/test/java/com/fasterxml/jackson/datatype/money/CurrencyUnitDeserializerTest.java

```diff
@@ -0,0 +1,43 @@
+package com.fasterxml.jackson.datatype.money;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.jsontype.BasicPolymorphicTypeValidator;
+import org.javamoney.moneta.CurrencyUnitBuilder;
+import org.junit.Test;
+
+import javax.money.CurrencyUnit;
+import javax.money.UnknownCurrencyException;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
+
+public final class CurrencyUnitDeserializerTest {
+
+    private final ObjectMapper unit = new ObjectMapper().registerModule(new MoneyModule());
```

Comment:
One minor request (apologies for forgetting to mention initially): let's use

    JsonMapper.builder().addModule(new MoneyModule()).build();

the reason begin that ObjectMappers are immutable in Jackson 3.0, so merging this module is easier if we use builder pattern.